pkg/operation/operation.go

```diff
@@ -420,56 +419,57 @@ func (o *Operation) ReportShootProgress(ctx context.Context, stats *flow.Stats)
 		description    = makeDescription(stats)
 		progress       = stats.ProgressPercent()
 		lastUpdateTime = metav1.Now()
+		shoot          = o.Shoot.Info
 	)
 
-	newShoot, err := kutil.TryUpdateShootStatus(ctx, o.K8sGardenClient.GardenCore(), retry.DefaultRetry, o.Shoot.Info.ObjectMeta,
-		func(shoot *gardencorev1beta1.Shoot) (*gardencorev1beta1.Shoot, error) {
-			if shoot.Status.LastOperation == nil {
-				return nil, fmt.Errorf("last operation of Shoot %s/%s is unset", shoot.Namespace, shoot.Name)
-			}
-			if shoot.Status.LastOperation.LastUpdateTime.After(lastUpdateTime.Time) {
-				return nil, fmt.Errorf("last operation of Shoot %s/%s was updated mid-air", shoot.Namespace, shoot.Name)
-			}
-			if description != "" {
-				shoot.Status.LastOperation.Description = description
-			}
-			shoot.Status.LastOperation.Progress = progress
-			shoot.Status.LastOperation.LastUpdateTime = lastUpdateTime
-			return shoot, nil
-		})
-	if err != nil {
+	if err := func() error {
+		patch := client.StrategicMergeFrom(shoot.DeepCopy())
```

Comment:
If you want to be consistent with your other changes. you can name it `statusPatch` instead of `patch`